app/models/hold.rb

```diff
@@ -81,6 +81,7 @@ def order_number
   end
 
   def send_admin_notification_email
+    return if Rails.env.development? || Rails.env.local?
```

Comment:
What is your thought process for not sending the e-mail in development and local? I'd preferably like to have a separate e-mail account for all environments. Or the same e-mail account for local/dev/qa. I believe production shouldn't be used in the lower environments.

Reply:
Yes makes sense to me.  @daryanypl may I pass this off to you so that you can decide what is best (not sending emails or creating new accounts to receive messages from different environments)?

Reply:
I like the idea of having separate email addresses to test with.  Lets accept this code as the immediate fix, and then add a Jira ticket to the backlog (I'll write it).